common/gtk-3.0/3.20/sass/_colors.scss

```diff
@@ -6,11 +6,18 @@
 }
 
 $base_color: if($variant =='light', #ffffff, #404552);
+
+$text_color: if($variant == 'light', #5c616c, #D3DAE3);
+$bg_color: if($variant =='light', #F5F6F7, #383C4A);
+$fg_color: if($variant =='light', #5c616c, #D3DAE3);
+$sp_menu: if($variant =='light', #dfdfdf, #2b2e39);
+
```

Comment:
Why are `$text_color` `$bg_color` and `$fg_color` declared here, while they are reset with original values right below?
If there's some purpose for this, why did you change the original color values for the light variant?

Also, I think lines `9`, `14` and `20` are unnecessary whitespace.